### DISCUSSION TITLE
EQ5 Check It Writing Question

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#339] [General]
Reviewer: @AbbyANoble 

So, I actually really like what we found when we were looking for assessment problems. I don't remember where this came from, but this is what we have in there:
<img width="581" alt="Screenshot 2024-06-13 at 1 56 58 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/798fac09-464a-410e-9fee-e7515f113ab4">

If we don't want this, what do we want to focus on? The outcome states, "Solve quadratic equations using factoring, the square root property, completing the square, and the quadratic formula and express these answers in exact form." There are a lot of different things in here, so do we want to focus on factoring and the quadratic formula (in exact form)?

**Comment by AbbyANoble** [comment]
Yeah, this is such a big objective with a lot of different skills!

I like this problem, but it feels more like an activity to me. We would need to come up with lots of different mistakes the student could make, or otherwise once you've found the trick then it's the same every time. It also feels like a higher

Could we reframe it as showing multiple quadratic equations that need solving and asking students to match which method makes most sense to use, with the caveat of not using the same method every time (so they don't quadratic formula everything)? Then they don't have to solve all of them, but maybe 2 out of 5 or something?

**Comment by tdegeorge** [comment]
So, something like this?

Task 1: There are many ways to solve a quadratic equation, but sometimes it is better to use one particular method. For each equation, determine which method would be the best to use and why. You must choose a different method for each equation and cannot use one method more than once. (All equations having rational or integer solutions?)

- quadratic equation with a=1 (factorable). Example: x^2+4x=-3
- quadratic equation with a binomial (square root property): Example: 10(t+6)^2+36=90
- quadratic equation with a=1 (complete the square - middle term can easily take half of and square): Example: a^2+14a=51
- quadratic equation with a not equal to 1 and not factorable (quadratic formula): 3x^2-8x=-4

Task 2: Choose two of the equations above and solve using the method you stated is the best to use. Do you still think that method was the best? Why or why not?

**Reply by AbbyANoble** [reply]
I like this!
